### PR TITLE
Service Case Patch For Hide All

### PR DESCRIPTION
**Pull request by CalebGerman**
Fixes:
https://dev.azure.com/bentleycs/ProductAdvancement/_workitems/edit/1570626

### CODE REVIEW

File: packages/changed-elements-react/src/api/ChangedElementEntryCache.ts [hunk 1]
```diff
@@ -451,6 +451,10 @@ export class ChangedElementEntryCache {
     return entries;
   }
 
+  public getChangeElementIdFromModelId(modelId: string) {
```
Comment (CalebGerman) [inline packages/changed-elements-react/src/api/ChangedElementEntryCache.ts:454]: Id's from model Id

Comment (CalebGerman) [inline packages/changed-elements-react/src/api/ChangedElementEntryCache.ts:454]: Add explicit return type and docs on function

File: packages/changed-elements-react/src/widgets/EnhancedElementsInspector.tsx [hunk 2]
```diff
@@ -1150,7 +1150,9 @@ export class ChangedElementsListComponent extends Component<ChangedElementsListP
     const visualizationManager = this.props.manager.visualization?.getSingleViewVisualizationManager();
     if (visualizationManager) {
       if (item.extendedData?.isModel) {
+        const set = this.props.manager.changedElementsManager.entryCache.getChangeElementIdFromModelId(item.id);
         await visualizationManager.toggleModel(item.id);
```
Comment (CalebGerman) [inline packages/changed-elements-react/src/widgets/EnhancedElementsInspector.tsx:1154]: remove optional chaining the type is narrowed from if statement

File: packages/changed-elements-react/src/widgets/EnhancedElementsInspector.tsx [hunk 3]
```diff
@@ -1150,7 +1150,9 @@ export class ChangedElementsListComponent extends Component<ChangedElementsListP
...
+        await visualizationManager?.toggleElementsVisibility(!isVisible, set, false, false);
```
Comment (roluk) [inline packages/changed-elements-react/src/widgets/EnhancedElementsInspector.tsx:1155]: Please add a changelog entry.

**Review by roluk — APPROVED** [review]

**Pull request merged**